<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphInteractionWindow</class>
 <widget class="QWidget" name="GraphInteractionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GraphInteractionWindow</string>
  </property>
  <widget class="QTextEdit" name="graphCommandsFrame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Введите  граф используя стандартные команды, разделяя их символом переноса строки(add_vertex v, add_edge v1 v2 label)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitGraphCommandsButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/submitIcon.png</normaloff>resources/submitIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="sequenceTypeFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите базовый тип последовательности  (ArraySequence/LinkedListSequence)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitsSequenceTypeButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/submitIcon.png</normaloff>resources/submitIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="dictionaryTypeFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите базовый тип словаря (AVLTreeDictionary/BTreeDictionary)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitDictionaryTypeButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/submitIcon.png</normaloff>resources/submitIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="drawGraphButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/boxIcon.png</normaloff>resources/boxIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="drawGraphLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Построить граф</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="drawSkeletonTreeButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/boxIcon.png</normaloff>resources/boxIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="drawSkeletonTreeLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Построить минимальное остовное дерево</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/quitIcon.png</normaloff>resources/quitIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>429</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="graphInteractionCommandsFrame">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите команды для изменения графа,  разделяя их символом переноса строки(add_vertex v, delete_vertex v, add_edge v1 v2 label, delete_edge v1 v2)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitGraphInteractionCommandsButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/submitIcon.png</normaloff>resources/submitIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="getDistanceButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/boxIcon.png</normaloff>resources/boxIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="getDistanceLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Получить расстояние между вершинами</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="distanceInfoFrame">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите две вершины (a,b)</string>
   </property>
  </widget>
  <zorder>drawGraphLabel</zorder>
  <zorder>drawSkeletonTreeLabel</zorder>
  <zorder>graphCommandsFrame</zorder>
  <zorder>submitGraphCommandsButton</zorder>
  <zorder>sequenceTypeFrame</zorder>
  <zorder>submitsSequenceTypeButton</zorder>
  <zorder>dictionaryTypeFrame</zorder>
  <zorder>submitDictionaryTypeButton</zorder>
  <zorder>drawGraphButton</zorder>
  <zorder>drawSkeletonTreeButton</zorder>
  <zorder>quitButton</zorder>
  <zorder>errorLabel</zorder>
  <zorder>graphInteractionCommandsFrame</zorder>
  <zorder>submitGraphInteractionCommandsButton</zorder>
  <zorder>getDistanceLabel</zorder>
  <zorder>distanceInfoFrame</zorder>
  <zorder>getDistanceButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
